UI/MapWnd.cpp

```diff
@@ -6119,19 +6119,19 @@ void MapWnd::ConnectKeyboardAcceleratorSignals() {
     hkm->Connect(this, &MapWnd::ShowMenu,               "map.menu",
                  new AndCondition(new VisibleWindowCondition(this), new NoModalWndsOpenCondition()));
     hkm->Connect(this, &MapWnd::KeyboardZoomIn,         "map.zoom_in",
-                 new AndCondition(new VisibleWindowCondition(this), new NoModalWndsOpenCondition()));
+                 new AndCondition(new VisibleMapWndCondition(*this), new NoModalWndsOpenCondition()));
```

Comment:
The signature is

````
VisibleWindowCondition(const GG::Wnd* tg)
````
Changing to *this would be passing a MapWnd& reference instead. What am I missing here?